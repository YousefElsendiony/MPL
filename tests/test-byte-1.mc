int main()
{
  /* Declarations of type byte */
  byte a;
  byte b;
  byte c;

 /* Declares an Array of size 3 */
  int[3] packetA;

  /* Assigns the Array Initial values */
  packetA = [3,2,1];

  prints("Example of Byte and Raw operation.");
  raw(15);
  raw(80);
  raw(255);
  raw(packetA);
  print($15);
  print($80);
  print($255);

  return 0;
}
