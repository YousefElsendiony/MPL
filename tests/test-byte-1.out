"Example of Byte and Raw operation."
0xf
0x50
0xff
0x3
15
80
255